packages/@aws-cdk/aws-ecs-patterns/test/fargate/load-balanced-fargate-service-v2.test.ts

```diff
@@ -183,6 +184,45 @@ describe('When Application Load Balancer', () => {
       ServiceName: 'myService',
     });
 
+    // ECS Exec
+    Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
+      PolicyDocument: {
+        Statement: [
```

Comment:
You could consider factoring out this statement, since its used twice in this file. And maybe also factor out all of the instances of this statement to a shared file in the unit tests directory.